src/client/components/results/MovieGrid.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Movie from './Movie';
+
+class MovieGrid extends React.Component {
+    constructor(props) {
```

Comment:
@vitkuz let's skip constructor in case you only call `super()`

Reply:
done